source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -5,4 +5,140 @@ Access Data From a Cursor
 .. default-domain:: mongodb
 
 Read operations that return multiple documents do not immediately return
-all values matching the query.
+all values matching the query. Because a query can potentially match
+large sets of documents, these operations rely upon a
+:java-docs:`FindIterable
+<apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html>`.
+
+A ``FindIterable`` iterates through all the documents in your collection
+to find documents that match your search criteria. It also allows you to
+further specify which documents to see by setting paramaters though methods. 
+
+This page uses an initiating method, ``find()`` to show how to access
+data from a ``FindIterable``. 
+
+Terminal Methods
+----------------
+
+Execute database operations on a ``FindIterable`` using terminal
+methods. Terminal methods return a single value after iterating thorugh
+all the documents in a collection.
+
+First
+~~~~~
+
+Use the :java-docs:`first() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoIterable.html#first()>`
+method to retireve the first document in your query results:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Cursor.java
+   :language: java
+   :dedent:
+   :start-after: begin firstExample
+   :end-before: end firstExample
+
+Into
+~~~~
+
+Use the :java-docs:`into() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoIterable.html#into(A)>`
```

Comment:
Consider adding something like:

"Most often this method is used when you know your query filter will return a small number of documents that will easily fit into available memory"